lua/class_definitions.lua

```diff
@@ -1518,46 +1518,13 @@ classes = {
     },
     mongroup = {
         attributes = {
-            posx = {
-                type = "int",
-                writable = true
-            },
-            posy = {
-                type = "int",
-                writable = true
-            },
-            posz = {
-                type = "int",
-                writable = true
-            },
-            tx = {
-                type = "int",
-                writable = false
-            },
-            ty = {
-                type = "int",
-                writable = false
-            },
-            dying = {
-                type = "bool",
-                writable = true
-            },
-            horde = {
-                type = "bool",
-                writable = true
-            },
-            diffuse = {
-                type = "bool",
-                writable = true
-            },
-            radius = {
-                type = "int",
-                writable = true
-            },
-            population = {
-                type = "int",
-                writable = true
-            }
+            pos = { type = "tripoint", writable = true },
+            target = { type = "tripoint", writable = false },
```

Comment:
No good reason, I just restyled it while I was changing it from a 3-tuple to a tripoint and left how it works alone.